armi/interfaces.py

```diff
@@ -153,6 +153,11 @@ def __init__(self, r, cs):
         self.cs = settings.getMasterCs() if cs is None else cs
         self.r = r
         self.o = r.o if r else None
+        # default interface variables used for tight coupling
+        self.tightCouplingTolerance = 1e-4
+        self.tightCouplingVariables = None
+        self.tightCouplingConvergeOn = None
```

Comment:
This might be better suited as `CLASS_ATTRIBUTES` since they are not really going to modified after the developer has set them up.

Reply:
https://github.com/terrapower/armi/pull/1033/commits/43f98e1f137252004f1ca16cb4c5eb62be006644